analysis/setup.py

```diff
@@ -0,0 +1,17 @@
+from setuptools import setup, Extension
+import pybind11
```

Comment:
Please add a setup script for any necessary compilation, e.g. a `setup_pybind.sh` file.

Reply:
solved, encapsuled in setup_pybind.sh file, please use chmod +x before using it.